oor_on_edge/settings/settings_schema.py

```diff
@@ -17,7 +17,7 @@ class AzureIoTSpec(SettingsSpecModel):
 class DataDeliveryPipelineSpec(SettingsSpecModel):
     detections_path: str
     metadata_path: str
-    ml_model_id: int
+    ml_model_id: str
```

Comment:
Why is this put back to a string again? I thought it has to be an int for the postprocessing infra.

Reply:
It doesn't work, because of the automatic parsing replacement it gets only parsed as a string 😞

Reply:
Ah OK, we just have to document it then to prevent making the same mistake again. Maybe we can put a conversion with check in the data delivery code?